<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeViewDialog</class>
 <widget class="QDialog" name="RecipeViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Рецепт</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog { background-color: #ffffff; }
QLabel { color: #212529; }
QLabel#nameLabel {
    font-weight: bold;
    font-size: 24px;
    color: #212529;
}
QLabel#timeLabel, QLabel#categoryLabel {
    font-size: 16px;
    color: #666;
}
QLabel#ingredientsTitleLabel, QLabel#stepsTitleLabel {
    font-weight: bold;
    font-size: 18px;
    color: #212529;
}
QPushButton#backButton, QPushButton#deleteButton, QPushButton#editButton {
    background: transparent;
    border: none;
    padding: 8px;
}
QPushButton#backButton:hover, QPushButton#deleteButton:hover, QPushButton#editButton:hover {
    background-color: #f8f9fa;
    border-radius: 8px;
}
QListWidget#ingredientsListWidget, QListWidget#stepsListWidget {
    border: 1px solid #e9ecef;
    border-radius: 12px;
    background-color: #f8f9fa;
    padding: 10px;
}
QListWidget#ingredientsListWidget::item {
    padding: 4px 8px;
    margin: 2px 0;
    background: white;
    border-radius: 4px;
    border: none;
    min-height: 24px;
}
QListWidget#stepsListWidget::item {
    padding: 12px;
    margin: 4px 0;
    background: white;
    border-radius: 8px;
    border: 1px solid #e9ecef;
}
QListWidget#stepsListWidget::item {
    margin-bottom: 12px;
    min-height: 80px;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <!-- Верхняя панель -->
   <item>
    <layout class="QHBoxLayout" name="topBarLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="margin">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/icons/arrow.png</normaloff>:/icons/icons/arrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="topSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="recipeTitleLabel">
       <property name="text">
        <string>Рецепт</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px; font-weight: bold; color: #212529;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="topSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/icons/free.png</normaloff>:/icons/icons/free.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/icons/edit.png</normaloff>:/icons/icons/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <!-- Основной контент -->
   <item>
    <layout class="QHBoxLayout" name="mainContentLayout">
     <property name="spacing">
      <number>30</number>
     </property>
     <property name="margin">
      <number>20</number>
     </property>
     <!-- Левая часть: фото, название, время, ингредиенты, категория -->
     <item>
      <layout class="QVBoxLayout" name="leftLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <!-- Фото -->
       <item>
        <widget class="QLabel" name="photoLabel">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 20px; background-color: #f8f9fa;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <!-- Название -->
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Название рецепта</string>
         </property>
        </widget>
       </item>
       <!-- Время -->
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string>Время приготовления: 0 минут</string>
         </property>
        </widget>
       </item>
       <!-- Ингредиенты -->
       <item>
        <widget class="QLabel" name="ingredientsTitleLabel">
         <property name="text">
          <string>Ингредиенты:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ingredientsListWidget">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
        </widget>
       </item>
       <!-- Категория -->
       <item>
        <widget class="QLabel" name="categoryLabel">
         <property name="text">
          <string>Категория:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="leftSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <!-- Правая часть: шаги приготовления -->
     <item>
      <layout class="QVBoxLayout" name="rightLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="stepsListWidget">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>RecipeViewDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>

